src/backend/joanie/settings.py

```diff
@@ -424,6 +424,19 @@ class Base(Configuration):
         environ_name="JOANIE_PAYMENT_SCHEDULE_LIMITS",
         environ_prefix=None,
     )
+    # The full list of countries available for use:
+    # https://github.com/workalendar/workalendar#available-calendars
+    JOANIE_CONTRACT_COUNTRY_CALENDAR = values.Value(
+        "workalendar.europe.France",
+        environ_name="JOANIE_CONTRACT_COUNTRY_CALENDAR",
+        environ_prefix=None,
+    )
+    # Number of days for the withdrawal period as required by your country's contract legislation
+    JOANIE_CONTRACT_WITHDRAWAL_PERIOD_DAYS = values.PositiveIntegerValue(
```

Comment:
Not quit sure that `CONTRACT` is useful.

`JOANIE_CONTRACT_COUNTRY_CALENDAR` -> `JOANIE_CALENDAR`
`JOANIE_CONTRACT_WITHDRAWAL_PERIOD_DAYS` -> `JOANIE_WITHDRAWAL_PERIOD_DAYS`

Reply:
True, you are right. Those setting variable are mostly used for the payment schedule, which is integrated into the contract. Plus, as you've mentioned, we might use those objects elsewhere in the application.